.gitmodules

```diff
@@ -4,3 +4,6 @@
 [submodule "externals/fmt"]
 	path = externals/fmt
 	url = https://github.com/fmtlib/fmt.git
+[submodule "faker-cxx"]
```

Comment:
why did you added library to its own dependencies? probably some mistake, please remove

Reply:
I don't know how this got there but I fixed it.